### PR TITLE
[4.x] Travis fixes

### PR DESCRIPTION
**Pull request by GrahamCampbell**
As you say, the good stuff is on master, but it would be useful if the builds were still at least working on 4.x. The code style run still writes a load of garbage to stderr, but at least it "works". :)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,5 @@
 language: php
 
-env:
-  global:
-    - COMPOSER_FLAGS="--no-interaction"
```
Comment (GrahamCampbell) [inline .travis.yml]: Travis already enables this by setting an environment variable for no interaction that composer sees.

**Comment by mvriel** [comment]
I will take a look at this tomorrow (or @jaapio, whoever is first ;)). I need some sleep in a bit :)

**Comment by GrahamCampbell** [comment]
Finally all green. 🚀

**Comment by mvriel** [comment]
Wasn't in bed yet :D Good work @GrahamCampbell ! Thanks!

**Comment by GrahamCampbell** [comment]
Haha. ;)

**Pull request merged**